tdm-be/src/controller/webhook.ts

```diff
@@ -19,7 +19,8 @@ router.post(
             return;
         }
 
-        enrichmentHook.success(id);
+        enrichmentHook.success(id).then(undefined);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Strange form. Is it to show that it's an asynchronous process?

Usually, I use an `async/await` syntax, especially when you do nothing in `then`.

Reply:
Express do not support async function, and we don't want to wait for enrichmentHook to finish. This function is use to get the result file from the processing.